k2/util/loguru/loguru.cc

```diff
@@ -0,0 +1,1875 @@
+#ifndef _WIN32
```

Comment:
Please indicate the origin of this file.

Reply:
Done.